lib/Auth.js

```diff
@@ -39,7 +39,7 @@ class Auth {
         const location = isInfo ?
             AuthConfig.LOCATION.ANY :
             (options.local ? AuthConfig.LOCATION.LOCAL : AuthConfig.LOCATION.GLOBAL);
-        this._helper = ImpCentralApiHelper.getEntity(options.endpoint, location);
+        this._helper = ImpCentralApiHelper.getEntity(options.endpoint, location, options);
```

Comment:
Check what's used from options in `getEntity` and replace it with account only if nothing else is used